mapbox_gl_platform_interface/lib/src/location.dart

```diff
@@ -38,7 +36,7 @@ class LatLng {
     return <double>[latitude, longitude];
   }
 
-  static LatLng _fromJson(dynamic json) {
+  static LatLng? _fromJson(dynamic json) {
```

Comment:
Rather than `dynamic`, it looks like `json` could be typed as a non-null `Map` and save a few null checks.

Reply:
Done.